app/code/Magento/Inventory/Indexer/ActiveTableSwitcher.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
```

Comment:
don't forget to use strict typing for newly created code

`declare(strict_types=1);`